helios-tools/src/main/java/com/spotify/helios/cli/command/ControlCommand.java

```diff
@@ -118,7 +118,7 @@ private boolean run(final Namespace options, final Target target, final PrintStr
                       final BufferedReader stdin)
       throws InterruptedException, IOException {
 
-    final HeliosClient client = Utils.getClient(target, err, username);
+    final HeliosClient client = Utils.getClient(target, err, username, options);
```

Comment:
I do agree though the Commands could be refactored to have easier access to all these params w/o passing them in every single `run()` though